### ISSUE TITLE
Supabase on Oracle ARM64

### ISSUE THREAD

**Issue by sneakyfluffy** [supabase/supabase#18815]
# Bug report

- [*] I confirm this is a bug with Supabase, not with my own application. 
- [*] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

When I want to self host Supabase with docker on Oracle VM ARM64 I get an error on supabase-vector container. 

Logs of supabase-vector:
<jemalloc>: Unsupported system page size
memory allocation of 5 bytes failed

After that nothing happens and starting of other containers is interrupted. 
## To Reproduce

1. Oracle OCI instance with ARM64 OS. (Rocky linux in my case)
2. Install docker
3. git clone supabase form github
4. copy .env file
5. docker compose up -d

## Expected behavior

Supabase instance should start in docker

## Screenshots

![image](https://github.com/supabase/supabase/assets/18366933/8d2e5055-a976-42a2-bc1e-74bd336b506f)

![image](https://github.com/supabase/supabase/assets/18366933/03cfd447-7803-41e1-8d56-ede560e36d74)

## System information

- OS: Rocky Linux 8.8 ARM64
- Oracle Cloud free tier

**Comment by kunasun** [comment]
Also having the exact same issue using Oracle Linux 8.8 ARM64, same error log message.

**Comment by mattyx97** [comment]
> Also having the exact same issue using Oracle Linux 8.8 ARM64, same error log message.
> 
> EDIT: I don't need the vector capabilities of Postgres for my current project so I removed the vector image from docker-compose and the vector attribute in the depends-on field under db in the docker-compose.yml file. This allows me to docker compose supabase without supabase-vector running as a stopgap fix for now

Please can you send me the fix?

**Comment by nicufarmache** [comment]
I'm having the same issue on Raspberry Pi OS 64bit

**Comment by ivasilov** [comment]
@Jayllyz @nicufarmache @mattyx97 This is probably due to `timberio/vector` not supporting your CPU architecture. 

To disable the vector container, just remove or comment the following lines https://github.com/supabase/supabase/blob/master/docker/docker-compose.yml#L348-L350 and https://github.com/supabase/supabase/blob/master/docker/docker-compose.yml#L384-L405.

**Comment by nns52k** [comment]
> I'm having the same issue on Raspberry Pi OS 64bit

same here

**Comment by rjbrown99** [comment]
FWIW - this helped.
https://github.com/raspiblitz/raspiblitz/issues/4346

TL;DR:
- On a Raspberry Pi, check your current page size with `getconf PAGE_SIZE`, which for me returned 16384
- Edit /boot/firmware/config.txt, add the line `kernel=kernel8.img` to switch to a kernel that uses 4K page sizes. Reboot.
- Run `getconf PAGE_SIZE` after the reboot and it should now show 4096
- Try supabase gain
